File: client/src/features/models/job.ts
export interface Organization{
    id: number;
    name: string;
    description: string;
    website: string;
}

export interface Job {
    id: number;
    name: string;
    description: string;
    SubmissionTime: string;
    available: boolean;
    organizationId: number;
    organization: Organization
}

File: API/Entities/Job.cs
using System;

namespace API.Entities
{
    public class Job
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime SubmissionTime { get; set; }

        public bool Available { get; set; }
        public Organization Organization { get; set; }

    }
}
